airbyte-integrations/connectors/source-snipeit/unit_tests/test_incremental_streams.py

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+
+from airbyte_cdk.models import SyncMode
+from pytest import fixture
+from source_snipeit.incremental_streams import Events
+
+
+@fixture
+def patch_incremental_base_class(mocker):
+    # Mock abstract methods to enable instantiating abstract class
+    mocker.patch.object(Events, "path", "v0/example_endpoint")
+    mocker.patch.object(Events, "primary_key", "test_primary_key")
+    mocker.patch.object(Events, "__abstractmethods__", set())
+
+
+def test_cursor_field(patch_incremental_base_class):
+    stream = Events()
+    expected_cursor_field = "updated_at/datetime"
+    assert stream.cursor_field == expected_cursor_field
+
+
+def test_get_updated_state(patch_incremental_base_class):
+    stream = Events()
+    # TODO: replace this with your input parameters
+    inputs = {
+        "current_stream_state": {
+            "updated_at/datetime": "2022-04-20 19:06:42.805431+08:00"
+        },
+        "latest_record": {
+            "updated_at": {
+                "datetime": "2022-04-20 19:07:23.181174+08:00"
+            }
+        }
+    }
+    expected_state = {"updated_at/datetime": "2022-04-20 19:07:23.181174+08:00"}
```

Comment:
The state is a nested field? Probably would be better transform and move to higher level if not it will break in normalization.